### ISSUE TITLE
eval IoU == 0

### ISSUE THREAD

**Issue by M0L4N** [MCG-NJU/SparseOcc#54]
python val.py --config configs/r50_nuimg_704x256_8f.py --weights checkpoints/sparseocc_r50_nuimg_704x256_8f_24e_v1.1.pth
Using /home/zxt/.cache/torch_extensions/py38_cu118 as PyTorch extensions root...
Detected CUDA files, patching ldflags
Emitting ninja build file /home/zxt/.cache/torch_extensions/py38_cu118/dvr/build.ninja...
Building extension module dvr...
Allowing ninja to set a default number of workers... (overridable by setting the environment variable MAX_JOBS=N)
ninja: no work to do.
Loading extension module dvr...
[2024-09-02 17:29:01,521][INFO] - Using GPU: NVIDIA GeForce RTX 4090
[2024-09-02 17:29:01,521][INFO] - Setting random seed: 0
[2024-09-02 17:29:01,521][INFO] - Loading validation set from data/nuscenes/
[2024-09-02 17:29:04,414][INFO] - Creating model: SparseOcc
[>>>>>>>>>>>>>>>>>>>>>>>>>>] 6019/6019, 23.6 task/s, elapsed: 255s, ETA:     0s
Starting Evaluation...
6019it [03:27, 29.00it/s]
+----------------------+----------+----------+----------+
|     Class Names      | RayIoU@1 | RayIoU@2 | RayIoU@4 |
+----------------------+----------+----------+----------+
|        others        |  0.000   |  0.000   |  0.000   |
|       barrier        |  0.000   |  0.000   |  0.000   |
|       bicycle        |  0.000   |  0.000   |  0.000   |
|         bus          |  0.000   |  0.000   |  0.000   |
|         car          |  0.000   |  0.000   |  0.000   |
| construction_vehicle |  0.000   |  0.000   |  0.000   |
|      motorcycle      |  0.000   |  0.000   |  0.000   |
|      pedestrian      |  0.000   |  0.000   |  0.000   |
|     traffic_cone     |  0.000   |  0.000   |  0.000   |
|       trailer        |  0.000   |  0.000   |  0.000   |
|        truck         |  0.000   |  0.000   |  0.000   |
|  driveable_surface   |  0.000   |  0.000   |  0.000   |
|      other_flat      |  0.000   |  0.000   |  0.000   |
|       sidewalk       |  0.000   |  0.000   |  0.000   |
|       terrain        |  0.000   |  0.000   |  0.000   |
|       manmade        |  0.000   |  0.000   |  0.000   |
|      vegetation      |  0.000   |  0.000   |  0.000   |
+----------------------+----------+----------+----------+
|         MEAN         |  0.000   |  0.000   |  0.000   |
+----------------------+----------+----------+----------+
[2024-09-02 17:37:51,746][INFO] - --- Evaluation Results ---
[2024-09-02 17:37:51,748][INFO] - RayIoU: 0.0000
[2024-09-02 17:37:51,749][INFO] - RayIoU@1: 0.0000
[2024-09-02 17:37:51,749][INFO] - RayIoU@2: 0.0000
[2024-09-02 17:37:51,749][INFO] - RayIoU@4: 0.0000

help plz

**Comment by YANG-CY-163** [comment]
There should be a logging info like `[INFO] - Loading checkpoint from path/to/pth` after `[INFO] - Creating model: SparseOcc` if the ckpt loads correctly. You can check if your path is correct.

**Comment by M0L4N** [comment]
solved thank you :)
